src/pricing/content/roof/overHang.js

```diff
@@ -0,0 +1,6 @@
+export const overHang = {
```

Comment:
I very liked the concept, but maybe we can put this in 1 file for each category, instead of a full directory for each category? I don't see it grows so much, and i think that would be better.